docs/docs/concepts/foundation/communication/public_private_calls/slow_updates_tree.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Privately access Historic Public Data
+---
+
+In Aztec, private and public execution environments are completely separate and operate with distinct state management. It is not possible for private functions to reliably access the most recent public data public state - only sequencers can do that.
+
+But, what about historical public data (or public data that changes infrequently)? Through a **slow updates tree**, private functions can access historical public state. Please note that we are still experimenting with this feature.
+
+On this page you will learn:
+
+1. Why a slow updates tree exists & use cases
+2. How it works
+3. How it can be used to access historical public data
+4. Limitations
+
+## The need for a slow updates tree
+
+This structure was created specifically to privately & publicly access historical public data that is not updated often. It should be used to store data that is not sensitive but needs to be accessible from private smart contracts. Using a slow updates tree, it is possible to:
+
+- Access historic public data from a private function
+- Access historic public data from a public function
+- Update public data (that does not need updated often) from public and private functions 
+
+This data structure is ideal for these use cases:
+
+- Address Registry: Enabling contracts to interact with other contracts more easily requires address storage accessible in both public and private executions. This can be particularly helpful in things such as proxy contracts.
+- Access Control: Managing privileges in contracts, such as a token contract owner’s ability to mint new tokens, is streamlined when control information is shared between public and private executions. This might include things like blacklists and whitelists.
+
+## How it works
+
+We developed the Slow Update Tree to help balance public and private execution in a blockchain context.
+
+Earlier systems typically used either fully public or entirely private state trees, which led to privacy leaks if reading public data from private state.
+
+Using a shared state tree, it is possible to privately access historic public data by providing a membership proof to show that a value is indeed part of a commitment, and then check that the commitment matches the one stored in the state, as shown below.
```

Comment:
im kinda using commitment to mean root. maybe i need to explain this better.

 a merkle tree is the current design but since we're talking about shared trees in general it doesnt have to be. to quote lasse - Generally, the design relies on vector commitments where the values and commitment are publicly available in such a manner that they can be easily read in public and used for inclusion checks in private. Logically, you can think of a contract specific SST as what is shown in Figure 1. Where we have a commitment Cm to a set of values Vm,j ∀j for a specific contract m. In the current design we are using Merkle Trees to commit the values.... We can provide a membership proof for Vm,j ∈Cm to show that the value is indeed part of the commitment, and then check that the commitment matches the one stored in the state